reana_job_controller/app.py

```diff
@@ -46,16 +54,110 @@ def filter_jobs(job_db):
 
 @app.route('/api/v1.0/jobs', methods=['GET'])
 def get_jobs():
+    """Get all jobs.
+
+    .. http:get:: /api/v1.0/jobs
```

Comment:
👍 for the PEP257 and the httpdomain docstring structure.  Shall you plug it into Sphinx (autodoc and friends) within this PR, or shall I do it after the merge?